lib/resultuploaders/s3compatible/s3compatible.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
+
+require 'aws-sdk-s3'
+require './lib/auxiliary/json_helper'
+
+# AutoHCK module
```

Comment:
Please remove this comment as it is redundant. Context: https://github.com/HCK-CI/AutoHCK/commit/7fe2b623c1b7ba9dbb310ba2f3ae7190039a9fab

Reply:
ok.